python/pyarrow/tests/test_fs.py

```diff
@@ -730,9 +726,10 @@ def test_get_file_info_with_selector(fs, pathfn):
             # + for s3fs 0.5.2 all directories are dropped because of buggy
             # side-effect of previous find() call
             # (https://github.com/dask/s3fs/issues/410)
```

Comment:
This comment is for old s3fs version we also don't test anymore, so let's remove it altogether
```suggestion
```